html/browsers/browsing-the-web/back-forward-cache/resources/executor.html

```diff
@@ -30,23 +31,31 @@
 
 window.isPageshowFired = false;
 window.isPageshowPersisted = null;
+
 window.loadCount = parseInt(localStorage.getItem(uuid + '.loadCount') || '0') + 1;
 localStorage.setItem(uuid + '.loadCount', loadCount);
 
 window.pageShowPromise = new Promise(resolve =>
-  window.addEventListener('pageshow', resolve, {once: true}));
-
-const executor = new Executor(uuid);
+  window.addEventListener('pageshow',
+                          async () => {
```

Comment:
I didn't fix this because I don't intend to land the test changes as part of this PR; they're mostly here to verify how it works in practice, so where it's more complex it doesn't seem worth spending too much time on at this stage.

Reply:
Aha, I'll ignore test changes then!